File: Javascript/Car.js
class Car {
    constructor(license, driver) {
        this.id;
        this.license = license;
        this.driver = driver;
        this.passenger;
    }
    printDataCar() {
        console.table(this.driver);
        console.log(this.driver.name);
        console.log(this.license);
    }
}

File: Python/car.py
from account import Account

class Car:
    id          = int
    license     = str
    driver      = Account
    passenger   = int

    def __init__(self, license, driver):
        self.license = license
        self.driver = driver

    def print_dates(self):
        print(f'El conductor {self.driver.name} tiene la placa {self.license} y el número de identificación {self.driver.document}')
